subgraphs/reviews/reviews.graphql

```diff
@@ -0,0 +1,36 @@
+schema @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@key", "@override", "@shareable"]) {
+  query: Query
+}
+
+type Product implements ProductItf @key(fields: "id") {
+  id: ID!
+  reviewsCount: Int!
+  reviewsScore: Float! @shareable @override(from: "products")
+  reviews: [Review!]!
+}
+
+interface ProductItf {
+  id: ID!
+  reviewsCount: Int!
+  reviewsScore: Float!
+  reviews: [Review!]!
+}
+
+type Query {
+  _service: _Service!
+  _entities(representations: [_Any!]!): [_Entity]!
+  Hi: String!
+}
+
+type Review {
+  id: Int!
+  body: String!
+}
+
+scalar _Any
```

Comment:
Does Strawberry automatically add these [Federation types](https://www.apollographql.com/docs/federation/federation-spec#federation-schema-specification).

Since these subgraph schema files are often published directly via `rover subgraph publish` they usually don't include the additional Federation types directly, similar to how `_service` removes the Federation types so the published schema (in Apollo Studio) will show a clean schema without the Federation plumbing.

Reply:
I've removed them for now, I'll make sure that we have a better way for exporting the federation schema to a file with strawberry in a future release

Reply:
@patrick91 sounds good. once `reviews.graphql` gets pushed up think we're good to land this.

Reply:
Forgot to push that! Done now 😊